autoload/slime.vim

```diff
@@ -138,7 +138,14 @@ endfunction
 function! s:WeztermConfig() abort
   if !exists("b:slime_config")
     let b:slime_config = {"pane_id": 1}
-  end
+  elseif exists("b:slime_config.pane_direction")
+    let tmp = system("wezterm cli get-pane-direction " . shellescape(b:slime_config["pane_direction"]), "\n")
```

Comment:
is the `\n` from STDIN necessary?

Reply:
I'll check, you are probably correct. I just copied from another part of the file.